crates/sui-deepbook-indexer/src/models.rs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use bigdecimal::BigDecimal;
+use diesel::data_types::PgTimestamp;
+use diesel::{Identifiable, Insertable, Queryable, Selectable};
+
+use sui_indexer_builder::{Task, LIVE_TASK_TARGET_CHECKPOINT};
+
+use crate::schema::{
+    balances, flashloans, order_fills, order_updates, pool_prices, progress_store, proposals,
+    rebates, stakes, sui_error_transactions, trade_params_update, votes,
+};
+
+#[derive(Queryable, Selectable, Insertable, Identifiable, Debug)]
+#[diesel(table_name = order_updates, primary_key(digest))]
+pub struct OrderUpdate {
+    pub digest: String,
+    pub sender: String,
+    pub checkpoint: i64,
+    pub package: String,
+    pub status: String,
```

Comment:
I highly recommend to use integer to represent status v.s. String (and we should fix this in bridge indexer). The former is very error prone